byzcoin/internal/parse.go

```diff
@@ -0,0 +1,139 @@
+package internal
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/BurntSushi/toml"
+	"go.dedis.ch/kyber/v3/suites"
+	"go.dedis.ch/kyber/v3/util/encoding"
+	"go.dedis.ch/onet/v3"
+	"go.dedis.ch/onet/v3/app"
+	"go.dedis.ch/onet/v3/log"
+
+	"go.dedis.ch/onet/v3/network"
+)
+
+// StrToSI converts a string to a ServerIdentity. If tls is true, it will se the URL to point
```

Comment:
This and the next function seem severely misguided; they are all about faking ServerIdentities where you don't have enough information to really know if you'll be able to talk to the server. It seems like they are only used for the Debug methods, which is fine. But then why can't they be private, and next to the debug methods? And have comments saying, "this isn't the right way to make a ServerIdentity, but it's a special case for debugging".